jspoon/src/main/java/pl/droidsonroids/jspoon/HtmlField.java

```diff
@@ -38,7 +38,7 @@
 
     private void setLocaleFromTag(String localeTag) {
         if (localeTag.equals(Selector.NO_VALUE)) {
-            locale = Locale.getDefault();
+            locale = Locale.ROOT;
```

Comment:
locale powinno być w 1. kolejności brane z headera `Accept-Language` (jeśli istnieje i jest poprawny). Jeśli tego źródła nie ma to chyba lepiej fallbackować się do `getDefault()` niż do `ROOT`. Dobrze pamiętam, że to służy do wyboru `Decimal/Number/DateFormat`u przy parsowaniu liczb, a te liczby siedzą w tekście w HTMLu? Jeśli tak to `ROOT` nawet ma mały sens, bo np. polski tekst z `,` zamiast `.` w ułamkach się nigdy nie sparsuje poprawnie.
Poza tym dobrze by było zrobić możliwość zapodania locale przez usera, ale to już raczej improvement na przyszłość.